lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -77,7 +77,7 @@
 
 AZURE_TAG_ARGS = dict(
     tags=dict(type='dict'),
-    append_tags=dict(type='bool', default=True),
```

Comment:
Please don't change the default value, otherwise it will break current behaviors

Reply:
Actually Azure modules accepting `append_tags` as parameter currently behave like if `append_tags` is always `False`. And this is what I understood from your previous comment https://github.com/ansible/ansible/pull/39577#discussion_r185974353. Please let me know if I completely misunderstood the meaning of your comment.

Reply:
Oh my bad, apologize, according the doc, it should be True by default